src/mem/external_master.hh

```diff
@@ -117,12 +126,15 @@ class ExternalMaster : public SimObject
   public:
     ExternalMaster(ExternalMasterParams *params);
 
-    /** Port interface.  Responds only to port "port" */
     Port &getPort(const std::string &if_name,
                   PortID idx=InvalidPortID) override;
 
-    /** Register a handler which can provide ports with port_type ==
-     *  handler_name */
+    /**
+     * Register a handler which can provide ports with port_type ==
```

Comment:
Can you provide any other context here? Do you know how this handler might be used?